library/aesni.h

```diff
@@ -51,6 +51,11 @@
 #define MBEDTLS_AESNI_HAVE_INTRINSICS
 #endif
 
+/* For 32-bit, we only support intrinsics */
+#if defined(MBEDTLS_ARCH_IS_X86) && (defined(__GNUC__) || defined(_MSC_VER))
+#define MBEDTLS_AESNI_HAVE_INTRINSICS
+#endif /* MBEDTLS_ARCH_IS_X86 */
```

Comment:
I don't think we need this comment on the `#endif` - not only is the `#endif` close enough to the `#if` that this just adds noise, the comment isn't (fully) correct. Best to remove it